tests/Integration/Behaviour/Features/Scenario/Product/add_product.feature

```diff
@@ -0,0 +1,17 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags add
+@reset-database-before-feature
+Feature: Add basic product from Back Office (BO)
+  As a BO user
+  I need to be able to add new product with basic information from the BO
+
+  @add
+  Scenario: I add a product with basic information
+    When I add product "product1" with following basic information in default language:
+      | name | bottle of beer |
+      | type | standard       |
+    Then product "product1" should have following values:
+      | active           | false          |
+      | condition        | new            |
+    And product "product1" type should be standard
+    And product "product1" name should be "bottle of beer" in default language
+    And product "product1" should be assigned to default category
```

Comment:
As suggested above, you should add a scenario to validate the the missing default language is not allowed

Reply:
And maybe one with an invalid type